cocos/dragon-bones/ArmatureDisplay.ts

```diff
@@ -998,14 +998,19 @@ export class ArmatureDisplay extends UIRenderer {
             if (!this._debugDraw) {
                 const debugDrawNode = new Node('DEBUG_DRAW_NODE');
                 debugDrawNode.hideFlags |= CCObjectFlags.DontSave | CCObjectFlags.HideInHierarchy;
-                const debugDraw = debugDrawNode.addComponent(Graphics);
-                debugDraw.lineWidth = 1;
-                debugDraw.strokeColor = new Color(255, 0, 0, 255);
-
-                this._debugDraw = debugDraw;
+                let debugDraw: Graphics | undefined;
+                try {
+                    debugDraw = debugDrawNode.addComponent('cc.Graphics') as Graphics;
+                    debugDraw.lineWidth = 1;
+                    debugDraw.strokeColor = new Color(255, 0, 0, 255);
+
+                    this._debugDraw = debugDraw;
+                    this._debugDraw.node.parent = this.node;
+                } catch (e: any) {
+                    errorID(4501, e.message as string);
+                    debugDrawNode.destroy();
```

Comment:
It is better to set debugDrawNode to null too.

Reply:
Fixed